analysis/report_templates/default.html

```diff
@@ -65,46 +65,84 @@ <h5 class="center-align brown-text text-darken-4">
+
+            {% elif experiment.benchmarks|length < 2 %}
+
+            No aggregate ranking as the data contains a single benchmark.
+
+            {% else %}
+
             {% block top_level_ranking %}
 
-            {% if experiment.rank_by_mean_and_average_rank.size < 2 %}
-            No ranking as the available data is only for a single fuzzer.
+            We show two different aggreage (cross-benchmark) ranking of fuzzers.
+            The first is based on the average of per-benchmarks scores, where
+            the score represents the percentage of the highest reached median
+            coverage on a given benchmark (higher value is better).
+
+            The second ranking shows the average rank of fuzzers, after we rank
+            them on each benchmark according to their median reached covereges
+            (lower value is better).
 
-            {% elif experiment.rank_by_mean_and_average_rank.size > 20 %}
-            Showing cross-benchmark ranking as a table, because the critical
-            difference plot currently only supports up to 20 fuzzers. The table
-            shows the average rank (lower better) of fuzzers after ranking them
-            on each benchmark according to the median reached coverages.
             <div class="row">
-                <div class="col s7 offset-s3">
-                    {{ experiment.rank_by_mean_and_average_rank.to_frame().to_html() }}
+                <div class="col s5 offset-s1">
+                    <h5 class="center-align">By avg. score</h5>
+                    {{ experiment.rank_by_median_and_average_normalized_score.to_frame().to_html() }}
+                </div>
+                <div class="col s5">
+                    <h5 class="center-align">By avg. rank</h5>
+                    {{ experiment.rank_by_median_and_average_rank.to_frame().to_html() }}
                 </div>
             </div>
 
-            {% elif experiment.benchmarks|length < 2 %}
-            No cross-benchmark ranking as the available data is only for a single benchmark.
+            <ul class="collapsible">
+                <li>
+                    <div class="collapsible-header">
+                        Critical difference diagram
+                    </div>
+                    <div class="collapsible-body">
+
+
+            {% if experiment.rank_by_median_and_average_rank.size > 20 %}
+
+            Too many fuzzers to render the diagram. The critical difference plot
+            currently only supports up to 20 fuzzers.
 
             {% else %}
-            Aggregate critical difference diagram showing average ranks when
-            ranking fuzzers on each benchmark according to their median reached
-            coverages. Critical difference is based on Friedman/Nemenyi post-hoc
-            test. See more in the <a href="https://google.github.io/fuzzbench/reference/report/">
-            documentation</a>.<br>
+
+            The diagram visualizes the second ranking while showing the
```

Comment:
Can be more explicit on what this second ranking is, in case, people want to copy this text with image ?

Reply:
Good idea. Done